plugin-k8s/alumet-user.yaml

```diff
@@ -0,0 +1,41 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  annotations:
+  name: alumet_cluster_role
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - pods/logs
+  - pods/status
+  - nodes
+  - namespaces
+  verbs:
+  - get
+  - list
+  - watch
+
+---
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: alumet-reader
+  namespace: default
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: alumet-cluster_role_binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: alumet_cluster_role
+subjects:
+  - kind: ServiceAccount
+    name: alumet-reader
+    namespace: default
```

Comment:
You need to do this on every namespaces you want to monitor isn't it ?

Reply:
As i can see, no, because, i can retrieve data from other namespace than default

Reply:
You mean on this cluster ?

![image](https://github.com/alumet-dev/alumet/assets/109792731/2d59012d-5e6d-463a-9987-df5d14d0e588)

🤡

Reply:
but created from the previous file, so how could it create all of these users ??